GETTING_STARTED.md

```diff
@@ -0,0 +1,106 @@
+# Getting started
+
+Firstly, start the application by running `python3 main.py`.
```

Comment:
It's `nexus-constructor.py` now, not `main.py`

Reply:
Or maybe they've downloaded the distributable executable?
Actually starting with downloading the exe may be a good idea, and just refer to the readme for someone who wants to run from source.